frontend/packages/dev-console/src/components/topology2/Topology.tsx

```diff
@@ -100,30 +103,70 @@ const Topology: React.FC<TopologyProps> = ({ data, serviceBinding }) => {
     };
   }, [selectedIds]);
 
+  React.useEffect(() => {
+    visRef.current.getGraph().setLayout(layout);
+  }, [layout]);
+
   const onSidebarClose = () => {
     setSelectedIds([]);
   };
 
   const renderControlBar = () => {
     return (
       <TopologyControlBar
-        controlButtons={createTopologyControlButtons({
-          ...defaultControlButtonsOptions,
-          zoomInCallback: action(() => {
-            visRef.current.getGraph().scaleBy(4 / 3);
-          }),
-          zoomOutCallback: action(() => {
-            visRef.current.getGraph().scaleBy(0.75);
-          }),
-          fitToScreenCallback: action(() => {
-            visRef.current.getGraph().fit(80);
-          }),
-          resetViewCallback: action(() => {
-            visRef.current.getGraph().reset();
-            visRef.current.getGraph().layout();
+        controlButtons={[
+          ...createTopologyControlButtons({
+            ...defaultControlButtonsOptions,
+            zoomInCallback: action(() => {
+              visRef.current.getGraph().scaleBy(4 / 3);
+            }),
+            zoomOutCallback: action(() => {
+              visRef.current.getGraph().scaleBy(0.75);
+            }),
+            fitToScreenCallback: action(() => {
+              visRef.current.getGraph().fit(80);
+            }),
+            resetViewCallback: action(() => {
+              visRef.current.getGraph().reset();
+              visRef.current.getGraph().layout();
+            }),
+            legend: false,
           }),
-          legend: false,
-        })}
+          {
+            id: 'colaLayout',
+            icon: (
+              <span
+                className={classNames('odc-topology__layout-icon', {
+                  'm-active': layout === COLA_LAYOUT,
+                })}
+              >
+                <TopologyIcon />1
+              </span>
+            ),
+            tooltip: 'Layout',
```

Comment:
@serenamarie125 we need better names than `Layout` and `Layout 2` no?

Reply:
I suggest 1 and 2, just as Kiali a does